Bugzilla/Template.pm

```diff
@@ -260,6 +260,129 @@ sub quoteUrls {
     return $text;
 }
 
+sub quoteUrls2 {
+    my ($text, $bug, $comment) = @_;
+    return $text unless $text;
+    my $anon_user = new Bugzilla::User;
```

Comment:
The indirect object syntax is a style that isn't recommended for new code,
so ``Bugzilla::User->new` is preferred.

are you sure this is a good way forward? I guess this means non-js clients at least get some tooltips for public bugs... and it doesn't hurt cache-ability.